docs/antctl.md

```diff
@@ -666,3 +667,29 @@ worker1 172.18.0.4 Alive
 worker2 172.18.0.3 Alive 
 worker3 172.18.0.2 Dead
 ```
+
+### Upgrade existing resources
+
+Antctl supports upgrading existing resources of CRDs managed by Antrea to the latest version. The related sub commands
```

Comment:
You may highlight that the command will only handle CRDs with suffix `crd.antrea.io`:
```suggestion
Antctl supports upgrading existing resources of CRDs with suffix `crd.antrea.io` to the latest version. The related sub commands
```

Reply:
Maybe simply say: antctl supports upgrading resources of Antrea CRDs to the latest version